src/scenes/Projects/Files/CreateProjectDirectory.tsx

```diff
@@ -17,9 +17,25 @@ export const CreateProjectDirectory = (
   props: Except<CreateProjectDirectoryProps, 'onSubmit'>
 ) => {
   const [createDirectory] = useCreateProjectDirectoryMutation();
-  const { project, directoryId } = useProjectCurrentDirectory();
+  const {
+    project,
+    directoryId,
+    loading,
+    canRead,
+  } = useProjectCurrentDirectory();
   const { enqueueSnackbar } = useSnackbar();
 
+  if (canRead === false) {
```

Comment:
Is this necessary? They can't even get to this code.

Reply:
Yeah, not really. But I had already implemented it before I updated to make it unreachable, so I just left it for thoroughness.